src/pages/Main/Home.js

```diff
@@ -77,7 +75,7 @@ const HomePage = (props) => {
           md={6}
           className="gutter-row right-conteiner"
         >
-          <Button onClick={() => i18n.changeLanguage(i18n.language == 'kn' ? "en" : "kn")}>{i18n.language == 'kn' ? "English" : "ಕನ್ನಡ"}</Button>
+          <Button type = "primary" className="orange-btn" onClick={() => i18n.changeLanguage(i18n.language == 'kn' ? "en" : "kn")}>{i18n.language == 'kn' ? "English" : "ಕನ್ನಡ"}</Button>
```

Comment:
I would suggest moving, the ternary code out of jsx. In the function do something like
const [currentLang, toggleLang] = language == 'en' ? ['en','kn'] : ['kn','en'];

Reply:
I have added as a const